lint.whitelist

```diff
@@ -827,3 +827,137 @@ MISSING DEPENDENCY: web-nfc/resources/nfc-helpers.js
 MISSING DEPENDENCY: shape-detection/resources/shapedetection-helpers.js
 MISSING DEPENDENCY: webxr/resources/webxr_util.js
 MISSING DEPENDENCY: contacts/resources/helpers.js
+
+# Tests that are false positives for using Ahem as a system font
+AHEM SYSTEM FONT: acid/acid3/test.html
+AHEM SYSTEM FONT: resource-timing/resources/all_resource_types.htm
+AHEM SYSTEM FONT: resource-timing/resources/iframe-reload-TAO.sub.html
+
+# The following are imported from mozilla-central and can't be modified in WPT.
+# They do load Ahem as a web font, but they use their own copy which trips the
+# lint rule. Basically false positives.
+AHEM SYSTEM FONT: css/vendor-imports/mozilla/mozilla-central-reftests/text3/segment-break-transformation-rules-003.html
```

Comment:
Since we don't control these tests and the lint isn't run where they are maintained, listing individual tests isn't too useful, and will probably just mean that this list has to be updated on each sync, without driving down the number of exceptions. I recommend a blanket css/vendor-imports/mozilla/mozilla-central-reftests/* here instead.

Reply:
I disagree with this. We should lint those tests like normal since they should be moved out of that directory and we don't want to pile up problems that have to be addressed during that move.

Reply:
That's fine with me, it's Mozilla that maintains this. I assumed the wpt lint wouldn't run when modifying these at the upstream location so that it'd just be a burden on @dbaron when syncing, but if that's fine, I'll send a revert of my simplification.

Reply:
https://github.com/web-platform-tests/wpt/pull/18929